.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
```

Comment:
Is this the right trigger?
Are we planning to do releases from this repo?

Reply:
The options we had are to build on tags or releases.In github,when you create a tag,it automatically creates release.SO have just put it to trigger on release

Reply:
I suggest changing this to a workflow_dispatch instead. So that we can generate the images manually, since we will be mostly rebase this repo.

Reply:
Changed the workflow to manual trigger